src/SystemServicesContext/useIsLocationNeeded.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState, useCallback } from "react"
+import { Platform } from "react-native"
+
+import useOnAppStateChange from "./useOnAppStateChange"
+import { doesDeviceSupportLocationlessScanning } from "../gaen/nativeModule"
+
+const useIsLocationNeeded = (): boolean => {
```

Comment:
would this hook be used outside the context of the `useIsLocationOn` hook? Could this become a two-way enum value like the exposure notifications for iOS?

Reply:
Oo I like that, what do you think of this @aledustet? https://github.com/Path-Check/gaen-mobile/pull/343/commits/6f7da2555cfa4afc39dc66276320b5e7f9312493